### PR TITLE
Upgrade to 2018 edition

### PR DESCRIPTION
**Pull request by VeaaC**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -6,4 +6,4 @@ pub use flatdata::{Archive, FileResourceStorage};
 
 mod osmflat;
 
-pub use osmflat::*;
+pub use crate::osmflat::*;
```
Comment (boxdot) [inline src/lib.rs:9]: Btw. I would prefer not to use *. In Python this is a total antipattern. And in the end we are not paying for the number of lines.

File: examples/render-svg.rs [hunk 2]
```diff
@@ -318,7 +313,7 @@ where
                 .expect("failed to write coordinates");
         }
 
-        let mut polyline = element::Polyline::new()
+        let polyline = element::Polyline::new()
```
Comment (boxdot) [inline examples/render-svg.rs:316]: Why this is not needed anymore? This was just a mistake, right?

Comment (VeaaC) [reply examples/render-svg.rs:316]: Guess so..interestingly it was fixed by cargo fix... :-)

File: src/lib.rs (hunk 1 above)
Comment (VeaaC) [reply src/lib.rs:9]: Here we are exporting everything in the generated file... seems to make a lot of sense?

**Pull request merged**